### ISSUE TITLE
ImportError: cannot import name 'normalizeEmojiString' from 'emojiNorm'

### ISSUE THREAD

**Issue by cvzi** [KOLANICH-tools/emojiSlugMappingGen.py#1]
When I try to run it I get:

```
ImportError: cannot import name 'normalizeEmojiString' from 'emojiNorm'
```

**Comment by KOLANICH** [comment]
Oops. I have forgotten to push the latest version into this repo. I have renamed that API into `normalizeEmoji` in latest `emojiNorm`.

I guess it should work by now.

Also you will need some dependencies. I don't use PyPi for them, they can be installed brom my GitHub account. In some future there going to be a package manager centered around GitHub repos.

It also has some limitations due to the flaw in https://github.com/KOLANICH-libs/emojiNorm.py (some emoji sequences require 2 presentation selectors in sequence (i.e. `"👩🏿\u200d❤\ufe0f\ufe0f\u200d👨🏻"`), `emojiNorm` strips that, since I assummed that that it is strange that such sequences exist. ), so use with care.
